sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -74,28 +80,47 @@ export const AppHeader = () => {
 
   return (
     <Header>
-      <Flex gap="10" as="nav" align="center">
+      <Flex justify="center" gap="10" as="nav" align="center">
         <Link href="/" passHref>
-          <a>
+          <a style={{ height: '29px' }}>
```

Comment:
After inspecting via dev tools, the issue with the alignment between the logo and "visit blog" button seemed to be caused by some default height on the `a` tag. Forcing a height on the `a` tag seems to be the only way to currently achieve alignment.

Let me know if you have any thoughts on how to better solve this problem!

Reply:
Just adding diplay flex to the `a` tag is working )